Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/QKFK-snap-rebuild-multiple-rep/snap-rebuild-multiple-rep

```diff
@@ -51,6 +55,57 @@ python3 Openshift-EE/utils/result/result_update.py $job_id QKFK 4-chaos "Check i
 exit 1;
 fi
 
+################
+# LitmusBook 2 #
+################
+
+run_id="snap-multiple-rep";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-liveness metadata=${run_id})
+echo $test_name
+
+cd litmus
+# copy the content of deployer run_litmus_test.yml into a different file to update the test specific parameters.
+cp apps/busybox/liveness/run_litmus_test.yml busybox_loadgen_snap_multiple_rep.yml
+
+# Update the environmental variables in litmus job spec.
+
+: << EOF
+  ---------------------------------------------------------------------------------------------------------------------
+ | specAttribute     | kind   |         default value               | test specifc value                               |
+  ---------------------------------------------------------------------------------------------------------------------|
+ | appNamespace      | env    | app-busybox-ns                      | snap-multiple-rep                                | 
+ | Litmus job label  | label  | liveness: litmus-busybox-liveness   | liveness: busybox-liveness-snapmultiplerep       |
+ | appLabel          | env    | app=busybox-sts                     | app=snap-multiple-rep                            | 
+  ----------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/value: app-busybox-ns/value: snap-multiple-rep/g' \
+-e 's/app=busybox-sts/app=snap-multiple-rep/g' \
+-e 's/liveness: litmus-busybox-liveness/liveness: busybox-liveness-snapmultiplerep/g' busybox_loadgen_snap_multiple_rep.yml
+
+cat busybox_loadgen_snap_multiple_rep.yml
+
+sed -i '/command:/i \
+          - name: RUN_ID\
+            value: '"$run_id"'\
+' busybox_loadgen_snap_multiple_rep.yml
+
+# Run the Litmus job and get the details of the litmus job from litmus_job_runner utils.
+bash ../Openshift-EE/utils/litmus_job_runner label='liveness:busybox-liveness-snapmultiplerep' job=busybox_loadgen_snap_multiple_rep.yml
```

Comment:
@harshita-sharma011 use the label name like `busybox-liveness-snap-multiple-rep`